src/Main.java

```diff
@@ -1,5 +1,52 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        for (int i = 1; i < 11; i++) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 2");
+        for (int i = 10; i >= 1; i--) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 3");
+        for (int i = 0; i < 18; i = i + 2) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 4");
+        for (int i = 10; i >= -10; i--) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 5");
+        for (int year = 1904; year < 2096; year = year + 4) {
+
+            System.out.println(year + "год является високосным.");
+        }
+        System.out.println("Задача 6");
+        for (int i = 7; i < 100; i = i + 7) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 7");
+        for (int i = 1; i < 513; i = i * 2) {
+            System.out.println(i);
+        }
+        System.out.println("Задача 8");
+        int Deposit = 29000;
```

Comment:
```suggestion
        int deposit = 29000;
```
названия переменных надо писать с маленькой буквы всегда